Week1/Section2/Exercise2/Source2.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+* Exercise 2 - Even or Odd
+Write a program that will ask the user for three numbers, then tell them whether the numbers are even or odd.
+
+Example output
+Please enter a value for number a: 5
+Please enter a value for number b: 6
+Please enter a value for number c: 99
+
+Number a is ODD!
+Number b is EVEN!
+Number c is ODD!
+
+*/
+
+#include <iostream>
+
+void evaluate(int value) {
```

Comment:
Code looks good! But try to make your functions explicitly state what they do. Of course, in a small program like this, it doesn't really matter, but we want to code as if we were working on a game project, so we need to reduce ambiguity. 

What does `evaluate()` do? It evaluates if a number is even or odd, so maybe name this `evaluateEvenOrOdd()`? But what does _evaluate_ mean? Does it save the result to a file? Does it wipe your HDD if the number is odd? Or does it print if the number is even or odd? In that case, maybe `printEvenOrOdd()`?

Reply:
okok this is a good advice thx